lang/English/modules/settings.php

```diff
@@ -45,7 +45,7 @@
 define('feed_url', "Feed URL");
 define('feed_url_info', "GrayCube.com is sharing a LGSL feed on the URL:<br><b>http://www.greycube.co.uk/lgsl/feed/lgsl_files/lgsl_feed.php</b>");
 define('charset', "Character Encoding");
-define('charset_info', "UTF8, ISO, ASCII, etc... Left it blank to use ISO encoding.");
+define('charset_info', "UTF8, ISO, ASCII, etc... Leave it blank to use ISO encoding.");
```

Comment:
Yes, I was not talking about the better wording, but what I mean is that this string is surely not true now, because it uses what is defined in the main lang file when empty, which is UTF-8. It will not use ISO encoding when empty but UTF-8 which is the standard we now use with Transifex<->OGP.